ZuluContent/Misc/CharacterCreation.cs

```diff
@@ -830,7 +830,7 @@ private static void AddSkillItems(SkillName skill, Mobile m)
                 }
                 case SkillName.Fletching:
                 {
-                    PackItem(new Board(14));
+                    PackItem(new Log(14));
```

Comment:
We should slate this and other character creation settings to be configurable too, like starting locs etc.

Reply:
yeah good idea, we need to also think about the starting equipment being abused - for example we used to macro new char creation and killing/logging out in an inn to gather resources :P